Website/src/rendering/src/pages/_app.tsx

```diff
@@ -8,13 +8,16 @@ import { CdpScripts, identifyVisitor } from '../services/CdpService'; // DEMO TE
 import { MerchandisingScripts } from 'src/services/MerchandisingService'; // DEMO TEAM CUSTOMIZATION - Discover integration
 import { config } from '@fortawesome/fontawesome-svg-core';
 import '@fortawesome/fontawesome-svg-core/styles.css';
+import reduxStore from '../redux/store';
 config.autoAddCss = false;
 
 // Using nprogress is completely optional.
 //  nprogress provides a loading indicator on page/route changes.
 // Remove it in package.json as well if removed here.
 import 'nprogress/nprogress.css';
 import 'assets/css/main.css';
+import OcProvider from 'src/redux/ocProvider';
```

Comment:
This file is from the JSS-Next.js boilerplate. We try to modify this file as least as possible to facilitate upgrades to future JSS releases.

For non-CSS imports, we group them together:
- Please move the 3 new import lines just below line 8 `import { MerchandisingScripts } from 'src/services/MerchandisingService'; // DEMO TEAM CUSTOMIZATION - Discover integration`.
- Please ensure the imports are using a relative path to the current file. This is usually required in this project because Storybook has issues with absolute links from the project root. Thus `'src/redux/ocProvider'` should be `'../redux/ocProvider'`
- Please also wrap them in a comment block to identify them as customizations on top of the JSS boilerplate code:
   ```typescript
   // DEMO TEAM CUSTOMIZATION - OrderCloud integration
   import { Provider } from 'react-redux';
   import reduxStore from '../redux/store';
   import OcProvider from '../redux/ocProvider';
   // END CUSTOMIZATION
   ```

Reply:
makes sense. updated